SpherePacking/MagicFunction/b/psi.lean

```diff
@@ -51,65 +51,468 @@ def ψT' (z : ℂ) : ℂ := if hz : 0 < z.im then ψT ⟨z, hz⟩ else 0
 
 end defs
 
- section eq
+section eq
 
 /- It is possible to express ψI, ψT, ψS in terms of `H`-functions directly. -/
 
 /- We first prove some auxiliary results. -/
 
 section aux
 
-private lemma slash_aux (z : ℍ) : (z ^ 2 : ℂ) = (z ^ (-2 : ℤ)) / ((z ^ (-2 : ℤ)) ^ 2) := by
-  have h₁ : (z : ℂ) ≠ 0 := by
-    rw [ne_eq, Complex.ext_iff]
-    push_neg
-    exact fun _ ↦ ne_of_gt z.2
-  symm; calc
-  _ = ((z : ℂ) ^ (-2 : ℤ)) / (z ^ (-4 : ℤ)) := by
-      congr 1
-      simp only [Int.reduceNeg, _root_.zpow_neg, inv_pow, _root_.inv_inj]
-      change ((z : ℂ) ^ 2) ^ 2 = z ^ (2 * 2)
-      exact (pow_mul (z : ℂ) 2 2).symm
-  _ = _ := by rw [← zpow_sub₀ h₁]; rfl
+private lemma z_nonzero (z : ℍ ): (z : ℂ ) ≠ 0 := by
+      simp only [ne_eq]
```

Comment:
Please could you make sure your contribution follows the usual Lean style guidelines, in particular with respect to indentation and spacing? You can see these here: https://leanprover-community.github.io/contribute/style.html#structuring-definitions-and-theorems and I have taken the liberty of adjusting the theorem statement here too:
```suggestion
private lemma z_nonzero (z : ℍ) : (z : ℂ) ≠ 0 := by
  simp only [ne_eq]
```

Reply:
@adomani Would you happen to know about this?

Reply:
This is probably referring to the `commandStart` linter.  Setting the linter option to `true` may be all that is needed.

Although it don't remember if I hooked it up into Anne's linterSet for mathlib.  I'll take a look tomorrow, but in the meatime setting the option either in the lakefile or (or whenever you want to use the linter) should suffice.

Reply:
I just randomly stumbled across this thread and checked: the `commandStart` linter is part of the mathlib linter set.
In other words, setting `linter.mathlibStandardSet` to `true` in your lakefile will also enable this linter. Hope that helps!

Reply:
Oh, thanks! If you want to make a PR enabling it (which might involve fixing the fallout), then I'd be very happy to merge it. But I realise this might be a bit of work for you, so no worries if not.

Reply:
I won't have time to do this in the next weeks, sorry.